src/types/ScholarshipAmount.ts

```diff
@@ -88,4 +79,12 @@ export default class ScholarshipAmount {
         return '(Unknown amount)';
     }
   }
+
+  intersectsRange(filterMin: number, filterMax: number): boolean {
```

Comment:
I would use `min` and `max` here instead as this function could potentially be used for non-filtering purposes.